modules/API/pages/built-in-endpoints.adoc

```diff
@@ -58,6 +60,10 @@ curl -X GET "http://localhost:9000/echo"
 
 === Health check (public)
 
+Note: 
+- This 'api/ping endpoint is depended on GUI service and it cannot run on the higher number of nodes than 3 based on https://github.com/tigergraph/cqrs/blob/900a0fb81a649adbb2d60884b2e5ad23bb43b2f1/tutopia/common/config/node_assignment.go#L16. Therefor, hitting this endpoint on those nodes > 3 shows 502 error. So, this should not be treated as "health check endpoint" which is confusing our customers: https://tigergraph.zendesk.com/agent/tickets/53725.
+Zulip thread: https://zulip.graphtiger.com/#narrow/stream/39-TigerGraph-Frontline/topic/.2353725.3A.20.5BGoldman.20Sachs-US.5D.20-.20Need.20help.20to.20setup.20the.20paral.2E.2E.2E 
+
```

Comment:
This again is an internal technical discussion. It seems there is some concern about the usefulness or reliability of the /api/ping endpoint. Please discuss internally to agree on under what conditions a user can safety use this endpoint and describe those conditions.